### PR TITLE
(MFR-40) Yuan ti human

### PR DESCRIPTION
**Pull request by BraabHimself**
Add yuan ti human race options to ToA

### CODE REVIEW

File: WotC material/v13/pub_20170919_ToA.js [hunk 1]
```diff
@@ -3,13 +3,85 @@ RequiredSheetVersion(12.999);
 // This file adds the backgrounds and beasts from the Tomb of Annihilation adventure book to MPMB's Character Record Sheet
 
 // Define the source
-SourceList.ToA={
```
Comment (Smashman) [inline WotC material/v13/pub_20170919_ToA.js:6]: Why was the source removed?

Comment (BraabHimself) [reply WotC material/v13/pub_20170919_ToA.js]: Woops! I was copying and pasting after moving stuff from the auto-gen files to the pub_ file and thought I had included the updated comment to the source.

Just pushed the fix.

**Comment by safety-orange** [comment]
This looks fine, although I have some ideas for improvements. Unfortunately, I also see some unrelated files being changed and that is causing a conflict with my recent changes. I'm not very familiar with GitHub, but I can't exclude those files from being merged. Would you mind making a pull request that only changes the files "pub_20161115_VGtM.js" and "pub_20170919_ToA.js"?

**Comment by safety-orange** [comment]
Thank you! I have merged it and immediately made some changes of myself to the racial variants (most importantly, added the `spellChanges` object so that animal friendship shows that it only affects snakes on the spell sheet.

**Pull request merged**